rdkit/DataStructs/test_list.py

```diff
@@ -3,9 +3,6 @@
   ("python", "UnitTestcBitVect.py", {}),
   ("python", "UnitTestBitEnsemble.py", {}),
   ("python", "UnitTestTopNContainer.py", {}),
-  ("python", "BitUtils.py", {}),
```

Comment:
This block of changes looks like you removed tests without replacing them anywhere. Did I miss something?